### ISSUE TITLE
dimensions

### ISSUE THREAD

**Issue by LucasSilvaFerreira** [CompVis/stable-diffusion#7]
Am I missing it?
Is it possible to use commands to increase the Width and Height?

**Comment by ChangelingSpy** [comment]
Use `--W` for width and `--H` for height in pixels
For example `txt2img.py --W 640 --H 480 --prompt ...`
To see the list of all arguments with descriptions use `--help`
Other places to find arguments with descriptions is either on the [README.md](https://github.com/CompVis/stable-diffusion#text-to-image-with-stable-diffusion) or directly in the [txt2img.py](../blob/main/scripts/txt2img.py#L45) file itself.

**Comment by asrlhhh** [comment]
I am hitting an error whenever I want to generate vertical aspect ratios like --H 640 --W 480:

RuntimeError: Sizes of tensors must match except in dimension 1. Expected size 16 but got size 15 for tensor number 1 in the list.

It seems that both W and H need to be a multiply of some number?

**Comment by Iuryck** [comment]
> I am hitting an error whenever I want to generate vertical aspect ratios like --H 640 --W 480:
> 
> RuntimeError: Sizes of tensors must match except in dimension 1. Expected size 16 but got size 15 for tensor number 1 in the list.
> 
> It seems that both W and H need to be a multiply of some number?

Yes I am having this error as well. Have you had success with any changes to height? I only had success with 512,512

**Comment by Iuryck** [comment]
Just had success with 640 by 640

**Comment by magnusviri** [comment]
[TL;DR: The model was trained on mostly 512x512 images, and performs best in regard to coherency at 512x512 generations.](https://www.reddit.com/r/StableDiffusion/comments/wuw9vq/oooook_stablediffusion_should_be_used_in_11_scale/)
